### ISSUE TITLE
custom needle 

### ISSUE THREAD

**Issue by shachardevops** [Martin36/react-gauge-chart#44]
![image](https://user-images.githubusercontent.com/36663539/80354294-320a9580-887f-11ea-81aa-b785473f2f91.png)  
it will be great if we could change the needle  to a different component

**Comment by dayrlism10** [comment]
do we have anything like this here?

**Comment by Martin36** [comment]
@dayrlism10 Nope, nothing like that is currently implemented. If anyone is up for it, feel free
